lib/from_openstudio/load/electric_equipment.rb

```diff
@@ -0,0 +1,64 @@
+# *******************************************************************************
+# Honeybee OpenStudio Gem, Copyright (c) 2020, Alliance for Sustainable
+# Energy, LLC, Ladybug Tools LLC and other contributors. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# (1) Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'honeybee/load/electric_equipment'
+require 'to_openstudio/model_object'
+
+module Honeybee
+    class ElectricEquipmentAbridged < ModelObject
+
+        def self.from_load(load)
+            # create an empty hash
+            hash = {}
+            hash[:type] = 'ElectricEquipmentAbridged'
+            # set hash values from OpenStudio Object
+            hash[:identifier] = clean_name(load.nameString)
+            unless load.displayName.empty?
+                hash[:display_name] = (load.displayName.get).force_encoding("UTF-8")
+            end
+            unless load.schedule.empty?
+                schedule = load.schedule.get
+                hash[:schedule] = schedule.name.to_s
```

Comment:
The schedule is a required attribute in honeybee-schema. So, if the schedule is ,empty?, then the resulting hash is invalid. I think we have two options for the case that there's no schedule:
1. Create a dummy "Always On" schedule and use that. I think we would implement this similar to how the `$heating_setpoint_schedule` should be implemented, which a check for .nil? before we try to create it.
2. Just perform a check for whether the schedule exists before we create the hash. If the schedule does not exist, then we just disregard the load. The check like this will be similar to the `unless electric_eq_def.wattsperSpaceFloorArea.empty?` that we currently perform before creating the load object.

I leave the decision up to you, @tanushree04 , about which one is easier to implement for you. But this current situation is going to produce hashes that are not serializable by the Honeybee Python library.

Reply:
We also need to make sure that the schedule is of an acceptable type. In other words, it has to be a ScheduleRuleset or a ScheduleFixedInterval. If it's not, then don't have translators for the schedule to the Honeybee hash so we'll have to either not translate the whole load object or we use an Always On schedule.